app/lib/evss/claims_service.rb

```diff
@@ -0,0 +1,33 @@
+module EVSS
```

Comment:
I think it would make sense to put services that interact with models in `app/services` and more abstract classes that could just easily be put into a rubygem inside `lib/<some_appropriate_namespace>` Not sure i've ever seen `app/lib/`